keadm/README.md

```diff
@@ -1,17 +1,258 @@
+                It is a mandatory flag
+
+**IMPORTANT NOTE:** The versions used for Docker, KubeEdge and K8S, should be same in both Cloud and Edge side.
+
+## Reset KubeEdge Cloud and Edge components
+
+Referring to `KubeEdge Installer Doc`, the command to stop KubeEdge cloud (edge controller). It doesn't uninstall/remove any of the pre-requisites.
+
+- Execute `sudo kubeedge reset`
+
+### Command flags
+
+```
+sudo kubeedge reset --help
+
+kubeedge reset command can be executed in both cloud and edge node
+In cloud node it shuts down the cloud processes of KubeEdge
+In edge node it shuts down the edge processes of KubeEdge
+
+Usage:
+  kubedge reset [flags]
+
+Examples:
+
+For cloud node:
+kubeedge reset
+
+For edge node:
+kubeedge reset --server 10.20.30.40:8080
+    - For this command --server option is a Mandatory option
+
+
+Flags:
+  -h, --help            help for reset
+  -s, --server string   IP:Port address of cloud components host/VM
+
+```
+
+## Simple steps to bring up a KubeEdge setup and deploy a pod
+**NOTE:** All the below steps are executed as root user, to execute as sudo user ,Please add **sudo** infront of all the commands
+### 1.EdgeController (With K8S API-Server)
+##### Install tools with the particular version
+```
+kubeedge init --kubeedge-version=<kubeedge Version>  --kubernetes-version=<kubernetes Version> --docker-version=<Docker version>
+```
+##### Install tools with the Default version version
+```
+kubeedge init --kubeedge-version= --kubernetes-version= --docker-version
+or
+kubeedge init --kubeedge-version --kubernetes-version --docker-version
+```
+
+**NOTE:**
+kubeadm join **192.168.20.134**:6443 --token 2lze16.l06eeqzgdz8sfcvh \
+         --discovery-token-ca-cert-hash sha256:1e5c808e1022937474ba264bb54fea42b05eddb9fde2d35c9cad5b83cf5ef9ac  
+         After Kubeedge init ,please note the **CloudIp** as highlighted above generated from console output and port is **8080**.
+
+### 2.Manually copy Certs.tgz. to /etc/kubeedge in Edge vm
+
+In Edge VM
+```
+mkdir -p /etc/kubeedge
+```
+In Cloud VM
+
+```
+scp -r certs.tgz username@ipEdgevm:/etc/kubeedge
+```
+
+In Edge VM untar the certs,tgz file
+
+```
+cd /etc/kubeedge
+tar -xvzf certs.tgz
+```
+
+### 3.Edge Node join
+##### Install tools with the Default version 
+```
+kubeedge join --kubeedge-version=<kubeedge Version>  --kubernetes-version=<kubernetes Version> --docker-version=<Docker version> --server=<CloudIp:8080>
+```
+
+```
+kubeedge join --kubeedge-version= --kubernetes-version= --docker-version= --server=CloudIp:8080
+or
+kubeedge join --kubeedge-version --kubernetes-version --docker-version --server=CloudIp:8080
+```
+**Note**:Cloud ip refers to Ip generated ,from the step 1 as highlighted 
+
+### 4.Node status on EdgeController console
+In the Cloud Vm run,
+```
+kubectl get nodes
+
+NAME             STATUS     ROLES    AGE     VERSION
+192.168.20.135   Ready      <none>   6s      0.3.0-beta.0
+ubuntu           NotReady   master   6m20s   v1.14.1
```

Comment:
please remove this line

Reply:
which line , Space ?? or status of kubect get nodes ?

Reply:
ubuntu node status please remove. This is not relevant to kubeedge setup

Reply:
ok